controllers/user.go

```diff
@@ -525,7 +525,23 @@ func deleteUser(w http.ResponseWriter, r *http.Request) {
 		logic.ReturnErrorResponse(w, r, logic.FormatError(err, "badrequest"))
 		return
 	}
-
+	// check and delete extclient with this ownerID
+	go func() {
+		extclients, err := logic.GetAllExtClients()
+		if err == nil {
```

Comment:
could invert if and reduce indention.
like having
```go
err != nil { log and return }
dostuff...
```

Reply:
done